docs/docs/sourcing-from-forestry.md

```diff
@@ -0,0 +1,179 @@
+use_front_matter_path: false
+file_template: ":filename:"
+build:
+  preview_output_directory: public
+  install_dependencies_command: npm install
+  preview_docker_image: node:10
+  mount_path: "/srv"
+  working_dir: "/srv"
+  instant_preview_command: npm run forestry:preview
+```
+
+The `new_page_extension` setting can either be `md` or `html`.
+
+Change `upload_dir` and `public path`:
+
+```yaml:title=.forestry/settings.yml
+upload_dir: static/uploads
+public_path: "/static/uploads"
+```
+
+This sets where media is saved. Visit Gatsby's [guide on its static folder](/docs/static-folder/) to learn more. The rest of the settings can remain the same.
+
+Open `package.json` and add the following to scripts:
+
+```json:title=package.json
+"scripts": {
+  "forestry:preview": "gatsby develop -p 8080 -H 0.0.0.0"
+},
+```
+
+For preview to work on Forestry's dashboard, port 8080 must be used and all network interfaces have to be bound to `0.0.0.0`. For more information on Forestry's previews go to the [Forestry.io docs](https://forestry.io/docs/previews/instant-previews/#adding-an-instant-preview).
+
+_Note_: The `forestry:preview` script can be named anything, but it must match the `instant_preview_command` setting in `.forestry/settings.yml`. This sets up Forestry to run a local server so that CMS content can be previewed from the Forestry dashboard using `gatsby develop`.
+
+### CMS Content Configuration
+
+Now you can set up a content collection. Create the file `.forestry/front_matter/blog.yml` and paste in the following:
+
+```yml:title=.forestry/front_matter/blog.yml
+---
+label: Blog
+hide_body: false
+fields:
+  - name: date
+    type: datetime
+    label: Date
+    description: ""
+    config:
+      required: true
+      date_format:
+      time_format:
+      display_utc: false
+    default: now
+  - name: title
+    type: text
+    label: Title
+    description: ""
+    config:
+      required: true
+```
+
+Next, link the frontmatter collection in `.forestry/settings.yml`. This adds a template in the Forestry dashboard that allows you to add new Blog posts with the configured fields in the CMS.
+
+```yml:title=.forestry/settings.yml
+---
+new_page_extension: md
+auto_deploy: false
+admin_path: /static/admin
+webhook_url:
+sections:
+upload_dir: static/uploads
+public_path: "/static/uploads"
+front_matter_path: ''
+use_front_matter_path: false
+file_template: ":filename:"
+build:
+  preview_output_directory: public
+  install_dependencies_command: npm install
+  preview_docker_image: node:10
+  mount_path: "/srv"
+  working_dir: "/srv"
+  instant_preview_command: npm run forestry:preview
+sections:
+- type: directory
+  path: src/content/posts
+  label: Posts
+  create: documents
+  match: "**/*"
+  templates:
+  - blog
+```
+
+Because Forestry's `admin.html` file is in the `/static` directory, this page will only be available once your Gatsby site is built. Deploy your site to Netlify and you can see your Forestry dashboard.
```

Comment:
Perfect! That solves it.

I'm actually wondering if we need the Netlify section? Users can accomplish the same test by running `gatsby build` and `gatsby serve` and then they have multiple options for deploying. Thoughts?

Reply:
Oh yes, good catch. That's way better. I was trying to figure out how to do that. I'll go ahead and change that. Also, it looks like a dependency of gatsby has a security vulnerability and is causing ci to fail. Do you know the fix for that?

Reply:
The team is on it. Nothing to do with your code specifically. We'll be good to merge when we get these final edits in. Thanks for your diligence with the tests though!